### PR TITLE
Automated 2 monitoring UI tests filter alert by status on alerts and silence page

### PR DESCRIPTION
**Pull request by lihongyan1**
Preflight run:
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/81597/
Test cases
https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-21131
https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-21108
Depend on PR:
https://github.com/openshift/cucushift/pull/7526

### CODE REVIEW

**Comment by lihongyan1** [comment]
@yapei @juzhao please help review

File: lib/rules/web/admin_console/4.1/common_ui_elements.xyaml [hunk 1]
```diff
@@ -72,6 +72,18 @@ set_input_value:
...
+    selector:
+      xpath: //input[contains(@class, '<class_text>')]
+    op: clear
+    type: input
+set_input_class:
+  element:
+    selector:
+      xpath: //input[contains(@class, '<class_text>')]
+    op: send_keys <input_value>
+    type: input
```
Comment (yapei) [inline lib/rules/web/admin_console/4.1/common_ui_elements.xyaml:86]: these changes look good

**Comment by liangxia** [comment]
@lihongyan1  There are conflicts.

**Comment by liangxia** [comment]
4.0 isn't a valid version now, and they are removed in master branch, please do not add them back when you resolve conflicts.

**Comment by lihongyan1** [comment]
> /hold

deleted the 4.0 rule file

File: lib/rules/web/admin_console/4.0/monitoring_alerts.xyaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
+click_icon_index_button:
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.0/monitoring_alerts.xyaml:1]: 4.0 rules should be removed.

File: lib/rules/web/admin_console/4.0/monitoring_metrics.xyaml [hunk 3]
```diff
@@ -0,0 +1,136 @@
+click_primary_menu_developer_mode:
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.0/monitoring_metrics.xyaml:1]: Same here.

**Comment by lihongyan1** [comment]
@liangxia deleted all the 4.0 related files

**Pull request merged**